
<html>
<head>

<style>
h1,h2 {color:darkred;
       background-color:white}
h3 {color:black; background-color:#CCF}

a {color: darkblue};
a:visited {color:maroon}
a:hover {color:darkblue}

</style>

<title> shinyGEO - GEO dataset analysis using R shiny  </title> </head>

<h2> shinyGEO: a web-based application for performing differential expression and survival analysis on Gene Expression Omnibus datasets </h2>
<h3> Summary </h3>
<p> Analyzing gene expression and correlating phenotypic data is an important method to discover insights about disease outcomes
 and prognosis. There are currently several web-based tools designed to address these analyses but are limited in usability, 
 data pipeline access, and reproducibility. We describe an R Shiny web application, <i>shinyGEO</i>, that can download gene 
 expression data sets directly from the Gene Expression Omnibus, and perform differential expression and survival analysis across selected genes and probes.
<p>

<h3> Citation </h3>
 
<p>Dumas J, Gargano MA, Dancik GM. <i>shinyGEO</i>: a web-based application for analyzing Gene Expression Omnibus datasets. Bioinformatics. 2016 Dec 1;32(23):3679-3681 (<a href = "http://bioinformatics.oxfordjournals.org/content/early/2016/08/07/bioinformatics.btw519.abstract">link</a>)</p>

<h3> Running shinyGEO </h3>
       
<p>For optimal performance, we recommend running <i>shinyGEO</i> locally by following the instructions at <a href = "https://github.com/gdancik/shinyGEO"> https://github.com/gdancik/shinyGEO </a> </p>

<p><i>shinyGEO</i> can also be run from the following sites:
<a href = "http://bioinformatics.easternct.edu/shinyGEO/"> http://bioinformatics.easternct.edu/shinyGEO/ </a> </p>
<a href = " https://gdancik.shinyapps.io/shinyGEO/">  https://gdancik.shinyapps.io/shinyGEO/ </a>
<p>
Interested in contributing to <i>shinyGEO</i>? Source code and local installation instructions are 
available from github:  <a href = "https://github.com/gdancik/shinyGEO"> https://github.com/gdancik/shinyGEO </a> 

<h3> Feedback </h3>
We love hearing from <i>shinyGEO</i> users. If you have questions, comments, or feature requests, please contact Garrett Dancik, 
<script language="JavaScript">
var user = "dancikg";
var at = "easternct.edu";
var linktext = user + "@" + at ;
document.write("<a href='" + "mail" + "to:" + user + "@" + at + "'>" + linktext + "</a>");
</script>

<h3> About </h3>
<i>shinyGEO</i> is a project of the <a href = "http://bioinformatics.easternct.edu">Bioinformatics Laboratory</a> 
at <a href = "http://www.easternct.edu">Eastern Connecticut State University</a>.
<ul>
<li> Main contributors: Jasmine Dumas, Michael Gargano*, Garrett M. Dancik, PhD
<li> Additional contributors: Ken-Heng Liao*, Gregory Harper*, Brandon Spratt*
</ul>
*, Computer Science major, Eastern Connecticut State University


<h3> Funding </h3>
This work was supported, in part, by Google Summer of Code funding to Jasmine Dumas in 2015.
<script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');

  ga('create', 'UA-62826724-2', 'auto');
  ga('send', 'pageview');

</script>


</body></html>



